src/filters/capture/affix.rs

```diff
@@ -1,20 +1,8 @@
 use crate::metadata::Value;
 
-use super::Metrics;
-
-fn is_valid_size(contents: &[u8], size: u32, metrics: &Metrics) -> bool {
-    // if the capture size is bigger than the packet size, then we drop the packet,
-    // and occasionally warn
-    if contents.len() < size as usize {
-        if metrics.packets_dropped_total.get() % 1000 == 0 {
-            tracing::warn!(count = ?metrics.packets_dropped_total.get(), "Packets are being dropped due to their length being less than {} bytes", size);
-        }
-        metrics.packets_dropped_total.inc();
-
-        false
-    } else {
-        true
-    }
+/// Returns whether the capture size is bigger than the packet size.
```

Comment:
It's really nice how clean this now - no more need to manually manage these metrics 👍🏻